pkg/test/masteruserrecord/master_user_record.go

```diff
@@ -189,19 +189,41 @@ func TargetCluster(targetCluster string) MurModifier {
 	}
 }
 
-func Account(cluster string, tier toolchainv1alpha1.NSTemplateTier) MurModifier {
+type AccountOption func(ua *toolchainv1alpha1.UserAccountEmbedded)
 
+func SyncIndex(index string) AccountOption {
+	return func(ua *toolchainv1alpha1.UserAccountEmbedded) {
+		ua.SyncIndex = index
+	}
+}
+
+// Account sets the first account on the MasterUserRecord
+func Account(cluster string, tier toolchainv1alpha1.NSTemplateTier, options ...AccountOption) MurModifier {
```

Comment:
May I know why you don't use the already existing type and functions?
There is already the `UaInMurModifier` type and related functions - is it possible to reuse them or transform the existing ones to your way? 
My point is that it looks like an overlap and duplication to me

Reply:
it's a `MurModifier` because I want to be able to do something like this:
```
murtest.NewMasterUserRecords(t, 10, "user-%d", murtest.Account("cluster1", oldNSTemplateTier))
```
where I create 10 MURs with an account using the given tier.

Reply:
I was refering to `AccountOption`

Reply:
> I was refering to `AccountOption`

Damn'! I always get confused with the block of code which is showed in the reviews 😬

Reply:
> There is already the `UaInMurModifier` type and related functions - is it possible to reuse them or transform the existing ones to your way?
> My point is that it looks like an overlap and duplication to me

ok, let me see what I can do about it.

Reply:
done in 8463a2b: using `UaInMurModifier` instead